CIScripts/Test/PesterLogger/RemoteLogCollector.ps1

```diff
@@ -125,12 +161,19 @@ function Merge-Logs {
         Write-Log ("=" * 100)
         Write-Log $ComputerNamePrefix
 
-        $Logs = $LogSource.GetContent()
-        foreach ($Log in $Logs.GetEnumerator()) {
-            $SourceFilenamePrefix = "Contents of $($Log.Key):"
-            Write-Log ("-" * 100)
-            Write-Log $SourceFilenamePrefix
-            Write-Log $Log.Value
+        foreach ($Log in $LogSources.GetContent()) {
```

Comment:
How does it work? You are iterating over LogSources in a loop that iterates over LogSources... A typo?